docs/modules/configuration/pages/dynamic-data-structure-configuration.adoc

```diff
@@ -1,138 +1,65 @@
-`add*Config()` method, the configuration may have been partially propagated to some
-cluster members and adding the configuration should be retried by the user.
-
-== Unsupported Configuration Methods
-
-Dynamic configuration is supported for all `add*Config()` methods except the following:
-
-* `SplitBrainProtectionConfig`: A new split-brain protection configuration cannot be dynamically added but other
-configuration can reference split-brain protections configured in the existing static configuration.
-* `WanReplicationConfig`: A new WAN replication configuration cannot be dynamically
-added, however existing static ones can be referenced from other configurations,
-e.g., a new dynamic `MapConfig` may include a `WanReplicationRef` to a statically
-configured WAN replication.
-* `ListenerConfig`: Listeners can be instead added at runtime via other API such as
-`HazelcastInstance.getCluster().addMembershipListener` and
-`HazelcastInstance.getPartitionService().addMigrationListener`.
-
-[[handling-configuration-conflicts]]
-== Handling Configuration Conflicts
-
-Attempting to add a dynamic configuration, when a static configuration for
-the same element already exists, throws `InvalidConfigurationException`. For example,
-assuming we start a member with the following fragment in `hazelcast.xml` configuration:
-
-[source,xml]
-----
-<hazelcast>
-    ...
-    <map name="sessions">
-        ...
-    </map>
-    ...
-</hazelcast>
-----
-
-Then adding a dynamic configuration for a map with the name `sessions` throws
-a `InvalidConfigurationException`:
-
-[source,java]
-----
-HazelcastInstance instance = Hazelcast.newHazelcastInstance();
-
-MapConfig sessionsMapConfig = new MapConfig("sessions");
-
-// this will throw ConfigurationException:
-instance.getConfig().addMapConfig(sessionsMapConfig);
-----
-
-When attempting to add dynamic configuration for an element for which dynamic
-configuration has already been added, then if a configuration conflict is detected
-a `InvalidConfigurationException` is thrown. For example:
-
-[source,java]
-----
-HazelcastInstance instance = Hazelcast.newHazelcastInstance();
-
-MapConfig sessionsMapConfig = new MapConfig("sessions").setBackupCount(0);
-instance.getConfig().addMapConfig(sessionsMapConfig);
-
-MapConfig sessionsWithBackup = new MapConfig("sessions").setBackupCount(1);
-// throws ConfigurationException because the new MapConfig conflicts with existing one
-instance.getConfig().addMapConfig(sessionsWithBackup);
-
-MapConfig sessionsWithoutBackup = new MapConfig("sessions").setBackupCount(0);
-// does not throw exception: new dynamic config is equal to existing dynamic config of same name
-instance.getConfig().addMapConfig(sessionsWithoutBackup);
-----
-
-To ignore conflicts between dynamic configurations, set the xref:ROOT:system-properties.adoc#ignore-dynamic-conf-conflicts[`hazelcast.dynamicconfig.ignore.conflicts`] system property to `true`.
+NOTE: Persistence for dynamic configuration is a beta feature.
```

Comment:
> Is this beta as in 5.1-BETA-version, or is it beta for 5.1 release?

As in 5.1-BETA-version, there is no reason it would be beta for 5.1 AFAIK.

> Also, whole dynamic configuration is currently implemented as EE feature

https://github.com/hazelcast/hz-docs/pull/336#discussion_r784021997